### PR TITLE
[11.0] [MIG] account_invoice_check_total: Migration to 11.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The same about squashing adjacent translation commits

**Comment by rven** [comment]
@pedrobaeza Do I need to squash "Launchpad automatic translations update." also, or only the "OCA Transbot updated translations from Transifex"

**Comment by pedrobaeza** [comment]
Both, please. I see also a lot of useless commits from the same author at the beginning that you can join by logical set. This way, we reduce the footprint of the repository.

**Review by angelmoya — CHANGES_REQUESTED** [review]
Tested on runbot, on account settings, "Check Total on Vendor Bills" is shown as Enterprise

**Comment by angelmoya** [comment]
settings odoo

**Comment by rven** [comment]
@angelmoya I removed the Enterprise label from the setting

**Review by adrian-pesol — APPROVED** [review]

File: account_invoice_check_total/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Add a Verification Total field on vendor bills.
+The user enters the taxes included invoice total as printed on the vendor bill, then enters the invoice lines and taxes. The system then checks the total computed by Odoo is the same as the verification total.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/10.0
```
Comment (feketemihai) [inline account_invoice_check_total/README.rst]: Can you update runbot version.

**Review by feketemihai — CHANGES_REQUESTED** [review]

File: account_invoice_check_total/README.rst (hunk 1 above)
Comment (rven) [reply account_invoice_check_total/README.rst]: @feketemihai Done!

**Review by feketemihai — APPROVED** [review]

**Review by angelmoya — APPROVED** [review]

**Pull request merged**